mason/search/features.mas

```diff
@@ -1,33 +1,138 @@
 <%args>
-  $maximum_export_size
+ $organisms         => undef
+ $feature_types     => undef
 </%args>
 
 <& /util/import_extjs.mas &>
```

Comment:
this mason (import_extjs.mas) is trying to pull in js and css that no longer exists. maybe remove this line and deprecate that mason file?